docs/solidity-by-example.rst

```diff
@@ -645,4 +645,499 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+Creating and verifying signatures
+=================================
+
+Signatures are used to authorize transactions,
+and they're a general tool that's available to
+smart contracts. In this chapter we'll use it to
+prove to a smart contract that a certain account
+approved a certain message. We'll build a simple
+smart contract that lets me transmit ether, but
```

Comment:
Using `we` and `I` at the same time might be confusing. Perhaps it would be better to introduce people like Alice and Bob?